arabic/dream/another-dream/It was a nice dream.md

```diff
@@ -0,0 +1 @@
+كنت أحاول الفرار من بعض المجرمين وبينما أركض تسائلت لماذا لا أواجه هؤلاء الأوغاد فاستدرت وواجهتهم حتى قضيت على أربعة رجال فى نصف دقيقة قبل أن ينادينى أحدهم بأن أسرع لأن ميعاد المدرسة سيفوتنى ان لم أستيقظ :) [nice dream](../another dream/It was a nice dream.md)
```

Comment:
It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.